README.md

```diff
@@ -2,67 +2,37 @@
 
 [![Release](https://github.com/modusbox/connection-manager-api/actions/workflows/releaseWorkflow.yml/badge.svg)](https://github.com/modusbox/connection-manager-api/actions/workflows/releaseWorkflow.yml)
 
+![Connection Manager Architecture](architecture.png)
+
+
 Connection Manager API is a component of the Mojaloop ecosystem that allows an administrator to manage the network configuration and PKI information for the Hub and a set of DFSPs.
 
 It provides a REST API, described using a [Swagger/OpenAPI document](./src/api/swagger.yaml).
 
-The current version uses both [cfssl](https://github.com/modusintegration/cfssl) and [openssl](https://www.openssl.org/) as the PKI engines which issue and process CSRs and Certificates. The specific version of cfssl that MCM depends on is kept in the [Dockerfile](./Dockerfile) as the value of the `branch` argument ( as in `--branch=v1.3.4` ) and can also be specified as an environment variable ( see `CFSSL_VERSION` below ).
-
 The API servers uses OAuth2 to implement security, as defined in the [OAuth2 implementation doc](./oauth2.md)
 
-## Running the server locally
-
-To run the server with all the defaults and no security, the simplest way is to run:
-
-```bash
-P12_PASS_PHRASE="choose your own password" npm start
-```
-
-The default config requires a `mysql` db running on the default port.
-
-Once running, you can access the [Swagger UI interface](http://localhost:3001/docs)
-
-## Running the server + db + web UI locally while developing
-
-The API server requires a mysql db. There's also a Web UI [https://github.com/modusbox/connection-manager-ui](https://github.com/modusbox/connection-manager-ui).
+## DFSP onboarding process using MCM
 
-To run them together, you can use the following setup:
+Here you can find a [document and video describing the onboarding processs using MCM](https://infitx.atlassian.net/wiki/spaces/CS/pages/18776713/SOP0009+-+HUB+PM4ML+-+On-boarding+Payment+Manager+v.0.1-20220307)
```

Comment:
Please take note that I believe this link to not be publically available.

We may need to either "migrate" it to another location using mark-down, or have the above possibly be made public.